bin/create-release

```diff
@@ -24,8 +24,8 @@ def git_generate_changelog
   end
 
   relevant_matches = matches.select do |match|
-    # ignore merges from staging; those represent DTTs
-    match['branch'] != 'staging'
+    # ignore merges from our various server branches
+    !match.nil? && !['staging', 'test', 'levelbuilder'].include?(match['branch'])
```

Comment:
We don't ignore them, IMO. It is often useful to know whether a scoop happened or not. In fact, it would be awesome to include timestamps (this would make it more helpful to non-eng, I'd guess).